ansible/playbooks/roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -7,10 +7,22 @@ node:
   redpanda:
     empty_seed_starts_cluster: false
     advertised_kafka_api:
-    - address: {{ hostvars[inventory_hostname].advertised_ip }}
```

Comment:
when only advertised ip is used, the cluster cannot talk amongst its members using their private ips and fails to start